### ISSUE TITLE
impossible to switch screenshot sound off

### ISSUE THREAD

**Issue by shalnoff** [mate-desktop/mate-utils#134]
It's really annoying shutter sound on screenshot. please make it optional by somehow. It would be better to remove it by default. IMHO It is generally bad and counter KISS idea to add anything except straight functionality for any *nix application. 

PS. when I click print-screen at 2am by mistake I was really scared by that sound. 
##

**Comment by raveit65** [comment]
It's really annoying that user don't post infos like distro or package version ;)
Sound events are disable by default from our side.
[rave@mother ~]$ gsettings get org.mate.sound event-sounds
false
I never enabled this settting because i like a quiet system, so i never heard that screenshots have a sound event.

**Comment by shalnoff** [comment]
raveit65, oops, sorry :) 
Release Linux Mint 17.3 Rosa 64-bit
Kernel Linux 3.19.0-32-generic x86_64
MATE 1.12.0

am I right understand that "gsettings get org.mate.sound event-sounds false" set all sounds off? but what if I'd like to switch it off just for single application?

**Comment by raveit65** [comment]
Yes, this disable all sound events, doing this for a single application isn't possible.
But i'm still wondering that you hear a sound.
Can you run mate-screenshot from a terminal to be shure that it isn't a app from cinnamon?
I know that cinnamon comes with a lot of sounds enable as default, which is annoying for me.

**Comment by shalnoff** [comment]
raveit65, yep, this is the application sound (mate-screenshot make is).
I'm running MATE, not Cinnamon (see above).

**Comment by zamicol** [comment]
This is a problem for me.  I would love a command line option to turn off the sound, or simple just always have it disabled.  

Also, I have `org.mate.sound event-sounds` disabled, but it still makes sounds.  

```sh
$gsettings get org.mate.sound event-sounds
false
```

I'm running i3

Version information: 
Package mate-utils:                              
i   1.26.0-0ubuntu1

Ubuntu 22.04.1 LTS
